routers/slack.router.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const router = express.Router();
+const { App } = require("@slack/bolt");
+const app = new App({
+  token: "xoxb-1273107934163-1285590347713-IFD6nYKd3E59NMKy3rdr1WSu",
+  signingSecret: "4c1f11f5986ab05b95a796b7897e87c3",
+});
+
+(async () => {
+  // Start your app
+  await app.start(3001);
+
+  console.log("⚡️ Bolt app is running!");
+})();
+
+// GET /api/recurringevents/
+router.get("/test", (req, res) => {
```

Comment:
As I understand this is a temporary endpoint. But let's use a more specific endpoint name like `/test/conversation`.